test/test_basics.jl

```diff
@@ -36,16 +35,25 @@ function test_padding(ndim)
     model = test_model(ndim; tti=false)
     modelPy = devito_model(model, Options())
 
-    v0 = sqrt.(1f0 ./model.m)
-    cut = remove_padding(deepcopy(modelPy.vp.data), modelPy.padsizes; true_adjoint=true)
-    vpdata = deepcopy(modelPy.vp.data)
+    m0 = model.m
+    mcut = remove_padding(deepcopy(modelPy.m.data), modelPy.padsizes; true_adjoint=true)
+    mdata = deepcopy(modelPy.m.data)
 
-    @show dot(v0, cut)/dot(vpdata, vpdata)
-    @test isapprox(dot(v0, cut), dot(vpdata, vpdata))
+    @show dot(m0, mcut)/dot(mdata, mdata)
+    @test isapprox(dot(m0, mcut), dot(mdata, mdata))
 end
 
 function test_limit_m(ndim, tti)
     model = test_model(ndim; tti=tti)
+
+    @test get_dt(model) == calculate_dt(model)
+    @test model[:m] == model.m
+    mloc = model.m
+    model.m .*= 0f0
+    @test norm(model.m) == 0
+    model.m = mloc
+    @test model.m == mloc
+
     srcGeometry = example_src_geometry()
     recGeometry = example_rec_geometry(cut=true)
```

Comment:
I know this line has not changed in this pull request but there are issues in this line when running the tests—i.e., `GROUP="All" julia --project test/runtests.jl`. Apparently `example_rec_geometry` is defined twice in [here](https://github.com/slimgroup/JUDI.jl/blob/twri/test/test_composite.jl#L5) and [here](https://github.com/slimgroup/JUDI.jl/blob/twri/test/utils.jl#L125), which to my understanding is causing the following error in testing:

```
Test basic utilities: Error During Test at /home/ali/.julia/dev/JUDI/test/test_basics.jl:114
  Got exception outside of a @test
  MethodError: no method matching example_rec_geometry(; cut=true)
  Closest candidates are:
    example_rec_geometry(; nsrc, nrec) at /home/ali/.julia/dev/JUDI/test/test_composite.jl:6 got unsupported keyword argument "cut"
  Stacktrace:
   [1] kwerr(::NamedTuple{(:cut,),Tuple{Bool}}, ::Function) at ./error.jl:157
   [2] test_limit_m(::Int64, ::Bool) at /home/ali/.julia/dev/JUDI/test/test_basics.jl:58
   [3] macro expansion at /home/ali/.julia/dev/JUDI/test/test_basics.jl:125 [inlined]
   [4] macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113 [inlined]
   [5] top-level scope at /home/ali/.julia/dev/JUDI/test/test_basics.jl:115
   [6] include(::String) at ./client.jl:439
   [7] macro expansion at ./util.jl:175 [inlined]
   [8] top-level scope at /home/ali/.julia/dev/JUDI/test/runtests.jl:52
   [9] include(::Module, ::String) at ./Base.jl:377
   [10] exec_options(::Base.JLOptions) at ./client.jl:288
   [11] _start() at ./client.jl:484
```

Reply:
Looks like some leftover duplicate indeed, removed from `test_composite.jl`